### ISSUE TITLE
GLU instead of ReLU

### ISSUE THREAD

**Issue by cfunk1210** [hanzhanggit/StackGAN-v2#4]
Why are you using gated linear units (GLU) instead of the ReLU mentioned in the paper? 

Also, why did you define your own GLU instead of using the built-in one?  Was there a reason or was it just not implemented yet when you wrote this?

Thanks in advance.

**Comment by hanzhanggit** [comment]
Yes. It is because that GLU was not implemented when we tried to utilize it.

**Comment by jppgks** [comment]
@hanzhanggit And what about the choice of GLU over ReLU?
